packages/web-components/src/components/va-notification/va-notification.tsx

```diff
@@ -0,0 +1,126 @@
+import { 
+  Component, 
+  Element,
+  Event,
+  EventEmitter,
+  Host, 
+  Prop, 
+  h 
+} from '@stencil/core';
+import classnames from 'classnames';
+
+/**
+ * @componentName Notification
+ * @maturityCategory dont_use
+ * @maturityLevel proposed
+ */
+
+@Component({
+  tag: 'va-notification',
+  styleUrl: 'va-notification.scss',
+  shadow: true,
+})
+export class VaNotification {
+  @Element() el!: any;
+
+  /**
+   * If `true`, the card will be visible.
+   */
+  @Prop() visible?: boolean = true;
+
+  /**
+   * Symbol indicates type of notification 
+   * Current options are: action-required, update
+   */
+  @Prop() symbol?: string = 'none';
+
+  /**
+   * Aria-label text for the close button.
+   */
+  @Prop() closeBtnAriaLabel?: string = 'Close notification';
+
+  /**
+   * If `true`, a close button will be displayed.
+   */
+  @Prop({ reflect: true }) closeable?: boolean = false;
+
+  /**
+   * If `false`, card will not have border
+   */
+  @Prop({ reflect: true }) hasBorder?: boolean = true;
+
+  /**
+   * If `false`, card will not have the word "CLOSE" next to close icon
+   */
+    @Prop() hasCloseText?: boolean = false;
+
+  /**
+   * Destination URL for link (optional)
+   */
+    @Prop() href?: string = '';
+
+  /**
+   * Text for destination link (optional)
+   */
+    @Prop() text?: string = '';
+  
+  /**
+   * Fires when the component is closed by clicking on the close icon. This fires only
+   * when closeable is true.
+   */
+  @Event({
+    composed: true,
+    bubbles: true,
+  })
+  closeEvent: EventEmitter;
+
+  private closeHandler(e: MouseEvent): void {
+    this.closeEvent.emit(e);
+  }
+
+  render() {
+    const {
+      visible,
+      symbol,
+      href,
+      text,
+      closeable,
+      hasBorder,
+      hasCloseText,
+    } = this;
+
+    if (!visible) return <div aria-live="polite" />;
+
+    const classes = classnames('va-notification', symbol, { 'has-border': hasBorder })
+
+    return (
+      <Host>
+        <va-card show-shadow="true">
+          <div class={classes} role="alert">
+            <i aria-hidden="true" role="img" class={symbol}></i>
+            <div class="body" role="presentation">
+              <slot></slot>
+              {(href && text) ? (
+                <va-link active href={href} text={text} />
+              ) : null}
+            </div>
+          </div>
+
+          {closeable && (
+            <button
+              class="va-notification-close"
+              aria-label={this.closeBtnAriaLabel}
+              onClick={this.closeHandler.bind(this)}
+            >
+              <i aria-hidden="true" class="fas fa-times-circle" role="presentation" />
+              {hasCloseText && (
+                <span>CLOSE</span>
```

Comment:
Is there a design reference for this close text? 
![Screenshot 2023-05-23 at 1 53 17 PM](https://github.com/department-of-veterans-affairs/component-library/assets/872479/bc24b9d1-e1ba-43fd-bb1e-fa4c4ebf3063)

Reply:
yes! @humancompanion-usds and @aagosto90 discussed building variations with the Close text: 

https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1181#issuecomment-1458632802

Reply:
@humancompanion-usds @aagosto90 @babsdenney The only Sketch file we have for QA'ing is this one I think: https://www.sketch.com/s/9b0e6efc-423a-4354-9db3-ab2083d566c9/a/ygvmgOq#Inspect

Do we happen to have any other source of truth or are we ok with using that one for now? There is no variation for the "close" text currently.

Reply:
@jamigibbs here you go - sorry about that these files includes the text and the x icon: [desktop](https://www.sketch.com/s/9b0e6efc-423a-4354-9db3-ab2083d566c9/a/ZOkzKM7), [mobile](https://www.sketch.com/s/9b0e6efc-423a-4354-9db3-ab2083d566c9/a/YG34bgd)

Reply:
@jamigibbs I updated the sketch file links in the body above as well